torch_frame/gbdt/tuned_lightgbm.py

```diff
@@ -226,3 +226,27 @@ def _load(self, path: str) -> None:
         import lightgbm
 
         self.model = lightgbm.Booster(model_file=path)
+
+    def _feature_importance(self, importance_type: str = 'gain',
+                            iteration: Optional[int] = None) -> list:
+        r"""Get feature importances.
+
+        Args:
+        importance_type (str): How the importance is calculated.
+            If "split", result contains numbers of times the feature
+            is used in a model. If "gain", result contains total gains
+            of splits which use the feature.
+        iteration (int, optional): Limit number of iterations in the feature
+            importance calculation. If None, if the best iteration exists,
+            it is used; otherwise, all trees are used. If <= 0, all trees
+            are used (no limits).
+
+        Returns:
+            list: Array with feature importances.
+        """
+        assert importance_type in [
+            'split', 'gain'
+        ], f'Expect split or gain, got {importance_type}.'
+        scores = self.model.feature_importance(importance_type=importance_type,
+                                               iteration=iteration)
+        return scores.tolist()
```

Comment:
Will this list to be just a list of scores? IMO it's better to return a dictionary where keys are column names and values are corresponding scores. WDYT?

Reply:
The return types of GBDT's feature importance API are different. For convenience, I converted them to lists.
```
lightgbm -> ndarray
xgboost -> dict[str, float]
catboost -> ndarray
```